src/styles/index.css

```diff
@@ -0,0 +1,117 @@
+.page {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.stars {
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4 {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--5 {
+  background-image: url(../images/star.svg);
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+  box-sizing: border-box;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+  margin-bottom: 5px;
+}
+
+.card__code {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 22px;
+}
+
+.card__reviews_stars {
+  display: flex;
+}
+
+.card__reviews_count {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__price_title {
+  color: #616070;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__price_sum::after {
+  content: "$2,199 ";
+  font-weight: 700;
+  color: #060b35;
+  line-height: 18px;
+}
+
+.card__button {
+  display: block;
+  box-sizing: border-box;
+  width: 100%;
+  height: 40px;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  background-color: #00acdc;
```

Comment:
The color values for the button's border and background are hardcoded. If these colors are used elsewhere, consider defining them as CSS variables for easier maintenance and consistency.